web-ui/src/main/resources/catalog/components/search/mdview/mdviewDirective.js

```diff
@@ -104,6 +104,17 @@
         },
 
         link: function(scope, element, attrs, controller) {
+          scope.isRatingEnabled = false;
+
+          var statusSystemRating =
+            gnConfig[gnConfig.key.isRatingUserFeedbackEnabled];
+          if (statusSystemRating == 'advanced') {
+            scope.isUserFeedbackEnabled = true;
```

Comment:
Is `scope.isUserFeedbackEnabled` used? I see it already defined in other code a similar property, but I guess is independent of the `gnMetadataRate`?:

- https://github.com/geonetwork/core-geonetwork/blob/3.4.x/web-ui/src/main/resources/catalog/components/search/mdview/mdviewModule.js#L60

- https://github.com/geonetwork/core-geonetwork/blob/3.4.x/web-ui/src/main/resources/catalog/js/search/SearchController.js#L63